core/field.js

```diff
@@ -45,7 +44,8 @@ goog.require('goog.userAgent');
  * @constructor
  */
 Blockly.Field = function(text, opt_validator) {
-  this.size_ = new goog.math.Size(0, Blockly.BlockSvg.MIN_BLOCK_Y);
```

Comment:
Another option is to leave the .size_ property in here and just turn it into an object with .width_ and .height_.  Disadvantage: extra indirection.  Advantage: not breaking alternative renderers (e.g. makecode)